BEST_PRACTICES.md

```diff
@@ -316,14 +294,6 @@ cont client: Client = new ClientBuilder()
 Requests can be made directly to the platform in different ways, here we will be seeing some of those ways requests can be made directly to the platform. Sometimes the SDK doesn't provide methods to include specific requests to the platform, in this case, we can manually construct the request and send it directly to the platform. In this situation, we can use the execute function to make this call.
 
 ```ts
-import { ClientBuilder } from '@commercetools/sdk-client-v2'
-
-cont client = new ClientBuilder()
-  .withClientCredentialsFlow(...)
-  .withHttpMiddleware(...)
-  ...
-  .build()
-
```

Comment:
Any specific reason why this code block was also removed ?

Reply:
yes, it's just redundant I added the client initialization at the top of the file